package.json

```diff
@@ -122,6 +122,7 @@
     "@types/puppeteer": "^5.4.4",
     "@typescript-eslint/eslint-plugin": "^5.11.0",
     "@typescript-eslint/parser": "^5.11.0",
+    "axios": "0.27.2",
```

Comment:
We can use node-fetch instead for tests. No need to bring in a new dependency :)